### ISSUE TITLE
python35Packages.breathe fails to build

### ISSUE THREAD

**Issue by siddharthist** [NixOS/nixpkgs#18945]
## Issue description
I get a traceback when installing `python35Packages.breathe`:
```
creating /tmp/nix-build-python3.5-breathe-4.2.0.drv-0/breathe-4.2.0/build/lib/breathe.egg-info
writing entry points to /tmp/nix-build-python3.5-breathe-4.2.0.drv-0/breathe-4.2.0/build/lib/breathe.egg-info/entry_points.txt
writing top-level names to /tmp/nix-build-python3.5-breathe-4.2.0.drv-0/breathe-4.2.0/build/lib/breathe.egg-info/top_level.txt
writing dependency_links to /tmp/nix-build-python3.5-breathe-4.2.0.drv-0/breathe-4.2.0/build/lib/breathe.egg-info/dependency_links.txt
writing requirements to /tmp/nix-build-python3.5-breathe-4.2.0.drv-0/breathe-4.2.0/build/lib/breathe.egg-info/requires.txt
writing /tmp/nix-build-python3.5-breathe-4.2.0.drv-0/breathe-4.2.0/build/lib/breathe.egg-info/PKG-INFO
writing manifest file '/tmp/nix-build-python3.5-breathe-4.2.0.drv-0/breathe-4.2.0/build/lib/breathe.egg-info/SOURCES.txt'
reading manifest file '/tmp/nix-build-python3.5-breathe-4.2.0.drv-0/breathe-4.2.0/build/lib/breathe.egg-info/SOURCES.txt'
writing manifest file '/tmp/nix-build-python3.5-breathe-4.2.0.drv-0/breathe-4.2.0/build/lib/breathe.egg-info/SOURCES.txt'
running build_ext
Traceback (most recent call last):
  File "nix_run_setup.py", line 8, in <module>
    exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\\r\\n', '\\n'), __file__, 'exec'))
  File "setup.py", line 79, in <module>
    use_2to3=True,
  File "/nix/store/52racbzwanwjdy256i0vjy2w1q0cr9lh-python3-3.5.2/lib/python3.5/distutils/core.py", line 148, in setup
    dist.run_commands()
  File "/nix/store/52racbzwanwjdy256i0vjy2w1q0cr9lh-python3-3.5.2/lib/python3.5/distutils/dist.py", line 955, in run_commands
    self.run_command(cmd)
  File "/nix/store/52racbzwanwjdy256i0vjy2w1q0cr9lh-python3-3.5.2/lib/python3.5/distutils/dist.py", line 974, in run_command
    cmd_obj.run()
  File "/nix/store/71cymngy86w0ml5ywj8vkbirlvwxai2d-python-3.5.2-bootstrapped-pip-8.1.2/lib/python3.5/site-packages/setuptools/command/test.py", line 172, in run
    self.run_tests()
  File "/nix/store/71cymngy86w0ml5ywj8vkbirlvwxai2d-python-3.5.2-bootstrapped-pip-8.1.2/lib/python3.5/site-packages/setuptools/command/test.py", line 179, in run_tests
    module = self.test_suite.split('.')[0]
AttributeError: 'NoneType' object has no attribute 'split'
builder for ‘/nix/store/np00d5xhazvma9ikpbmm6zh3afdf5bm0-python3.5-breathe-4.2.0.drv’ failed with exit code 1
error: build of ‘/nix/store/np00d5xhazvma9ikpbmm6zh3afdf5bm0-python3.5-breathe-4.2.0.drv’ failed
```
The issue is not present with `python27Packages.breathe`. 

### Steps to reproduce
```
nix-env -f '<nixpkgs>' -iA python35Packages.breathe
```

## Technical details

* System: 17.03.git.da514a2 (Gorilla)
* Nix version: nix-env (Nix) 1.11.4
* Nixpkgs version: 17.03.git.da514a2M

**Comment by FRidh** [comment]
This error often happens when the package is Python 2.x only. Judging from the trove classifiers this package is indeed only supported on Python 2.x. Therefore, we should set `disabled = isPy3k;`

**Comment by FRidh** [comment]
Fixed with a441efcccce769f39714ca008ee1e77682ae43a2 and c45fc1371e593d5721632d012006cf71021bda62

**Comment by siddharthist** [comment]
Thanks @FRidh! I'll keep that in mind if I see this one again.
